### PR TITLE
Goals Capture: Enable goals step in Production environment.

### PR DESCRIPTION
**Pull request by phcp**
#### Changes proposed in this Pull Request

This PR enables `signup/goals-step` feature flag on Production environment.

### CODE REVIEW

File: config/development.json [hunk 1]
```diff
@@ -144,7 +144,7 @@
 		"signup/design-picker-premium-themes-checkout": true,
 		"signup/design-picker-generated-designs": true,
 		"signup/inline-help": false,
-		"signup/goals-step": false,
+		"signup/goals-step": true,
```
Comment (yansern) [inline config/development.json]: This should go away after a rebase as it is enabled in the Test PR (for tests to work).

Comment (phcp) [reply config/development.json]: Rebasing it

Comment (phcp) [reply config/development.json]: Done!

**Review by yansern — APPROVED** [review]
Merge when Goals Capture is ready to land on production.

**Pull request merged**